### ISSUE TITLE
I can't get the best result after training

### ISSUE THREAD

**Issue by XiwuChen** [iMoonLab/MeshNet#8]
I tried to train this model from scratch but failed to get the accuracy of 91.92%. I only get the best model with Acc of 89.16% and mAP of 79.81%. I used the configuration and dataset you provided. Look forward to your help.

~~~yaml
# CUDA
cuda_devices: '3'   # multi-gpu training is available

# dataset
dataset:
  data_root: 'ModelNet40_MeshNet/'
  augment_data: true
  max_faces: 1024

# result
ckpt_root: 'ckpt_root/'

# MeshNet
MeshNet:
  structural_descriptor:
    num_kernel: 64
    sigma: 0.2
  mesh_convolution:
    aggregation_method: 'Concat'  # Concat/Max/Average

# train
lr: 0.01
momentum: 0.9
weight_decay: 0.0005
batch_size: 64
max_epoch: 500
milestones: [30, 60]
gamma: 0.1
~~~

**Comment by JsBlueCat** [comment]
> 91.92%.

i get always the same accuarcy  0.0418

**Comment by thss15fyt** [comment]
@XiwuChen Sorry for the late reply. The best result I got was trained from a more complex training strategy, which has not been organized into an automated script. And the result may be slightly different when you train several times. (This could be caused by the problems in the dataset.)

**Comment by vinitveersingh** [comment]
Hello @thss15fyt,
Thank you for your work.
I trained your model and I'm not able to recreate the test accuracy of the pre-trained model. I'm getting a test accuracy of 0.8847 after 150 epochs and a training accuracy of  0.9054. 

Can you tell me what was this complex training strategy or share its code?
Also, what problem in the data set are you referring to?

Thank you.
